### DISCUSSION TITLE
How to access Administration Console  of keycloak  from aws ec2 instance

### DISCUSSION THREAD

**Discussion by nikuldev** [Sunbird-RC/community#379] [Questions (FAQs)]
Hello,
I did the sunbird-rc setup on aws-ec2 ubuntu instance.
I want to get token from keycloak which is required to access the Sunbird APIs.
But I am getting HTTPS required on UI while accessing the admin portal of keycloak.
are there any changes required?

![Keycloak-network](https://user-images.githubusercontent.com/127303530/224017149-18e6d911-d999-40f0-9462-59de79a69f08.PNG)

![keycloak](https://user-images.githubusercontent.com/127303530/224017193-cac584cc-c679-469f-82f0-30f152782d7e.PNG)

**Comment by ddevadat** [comment]
i also faced the same issue when installed on a developer setup via docker compose.

I updated the backed postgres db for keycloak and after that i was able to login. Not sure if this is the right way to do it.
`update REALM set ssl_required='NONE' where id = 'master'`
